functions/menu.bash

```diff
@@ -186,15 +200,18 @@ show_main_menu() {
     "67 | Log Viewer"             "The openHAB Log Viewer webapp (frontail) " OFF \
     "68 | FireMotD"               "Configure FireMotD to present a system overview on SSH login (optional) " OFF \
     "69 | Bash&Vim Settings"      "Apply openHABian settings for bash, vim and nano (optional) " OFF \
-    "6A | Use ZRAM"               "Use compressed RAM/disk sync for active directories to avoid SD card corruption" OFF \
+    "6A | Use ZRAM"               "Use compressed RAM/disk sync for active directories (reduces SD card wear)" OFF \
```

Comment:
not Java PR related

reduces -> mitigates

Reply:
That was so the menu would fit without extending the width requirements.

Reply:
ok but I dislike "reduces", that ain't strong enough.

Reply:
changed, I also opened a PR to add the little unrelated fixes to master outside of this one.